README.md

```diff
@@ -1,2 +1,52 @@
 # ft_transcendence
-🏓 Multiplayer pong tournaments
+🏓 Play pong online with your friends in real time
+
+# 🤝 Contributing
+
+If you want to contribute to this project you may follow a set of rules.
+
+### 1. Create a new issue or task:
+
+- With every contribution there must be a related issue/task, so if you
+don't have any, create a new one on the project board with some
+detailed info about what do you want to accomplish.
+
+### 2. Create a new branch:
+   
+- ⚠️ We **do not** push to `master`, never. So you'll need to create a branch
+to work on it.
+- Try to start always from `master` and do not forget to bring
+all new changes, preferably with `git pull --rebase` so you don't corrupt
+commit history.
+
+### 3. Naming:
+
+- What about branch naming? A good standard way to name could be your
+GH userName + brief description of the task. For example:
+`marvin/add-contributing-docs`
+- We also try to have a **normalized way to name commits**. Because is complicated
+to always have great commit messages (despite you should) a good enough 
+naming strategy is adding your issue number followd by the message. This way other devs can
+keep track of some context of the changes just visiting the related issue
+because GitHub will link to it. Try something like:
+`#11 / add main contributing points`
+
```

Comment:
When talking about comits, sometimes I just push a commit to the working branch to save my progress, saying WIP or something like that. Should I say '#11 / WIP: comments to this PR' or something like that?

Reply:
In that case, what I usually do is amend the commit and then force push. I believe having a commit named "WIP" doesn't add much value... Something like:
1. Commit with the name you'd use if task was finished.
2. Next time, `git commit --amend --cleanup=whitespace` (`--cleanup=...` option is due to issues starting with `#` are treated as comments by git so we want to avoid this) Then an editor will open, remove everything but the commit message before saving)
3. And then, `git push --force`

⚠️ Again, this is my usual way of working, if someone has another opinion please manifest yourself!!

Reply:
praise: I like this, I think it makes sense to add the issue number to the commit message, and it's a great way to relate commits to issues.

Reply:
As talked yesterday with @Damrod, maybe we don't feel very comfy with force pushing and we want to have some freedom in our branch commits. He proposed a nice idea which is:
1. total freedom in how we make our commits but, when integrating, we should always use a `squash` strategy with the format -> `issue number + meaningful message for the task`
2. The PR should be associated with a reasonable size task so we don't get a huge commit

We all agree with this? If so, I'll update